client/src/Components/Header/AnonymousDiv.tsx

```diff
@@ -1,11 +1,66 @@
-import { LogInput, LogDiv, LogButton } from './AnonymousDivStyles';
+import { Axios } from '../../helpers/axios';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { useToasts } from 'react-toast-notifications';
+
+import { LogInput, LogDiv, LogButton, ErrorMessageEmail, ErrorMessagePassword } from './AnonymousDivStyles';
+
+interface Values {
+  email: string;
+  password: string;
+}
+
+const SignupSchema = Yup.object().shape({
+  email: Yup.string().email('Niewłaściwy adres email').required('Email jest wymagany!'),
+  password: Yup.string().min(8, 'Hasło musi zawierać conajmniej 8 znaków!').required('Hasło jest wymagane!'),
```

Comment:
Zrezygnowaliśmy z walidacji czy jest cyfra?

Reply:
niee, tylko zapomniałam to w końcu dodać :( wrzyciłam w tym commicie:
https://github.com/maciekkwiet/Bookweb/pull/81/commits/3a4a4db9a146f17e414294bbde9e5b133bcd3aa9